Robot.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.TimedRobot;
+
+/**
+* The VM is configured to automatically run this class, and to call the functions corresponding to
+* each mode, as described in the TimedRobot documentation. If you change the name of this class or
+* the package after creating this project, you must also update the build.gradle file in the
+* project.
+*/
+public class Robot extends TimedRobot {
+    private Compressor mCompressor;
+    private DoubleSolenoid mSolenoidOne;
+    private DoubleSolenoid mSolenoidTwo;
```

Comment:
Can you all your changes to the Robot.java in `src/main/java/frc/robot/Robot.java`? As of right now, none of this code would run because it is not in the real Robot.java file.